sda-commons-server-cloudevents/src/main/java/org/sdase/commons/server/cloudevents/CloudEventV1.java

```diff
@@ -0,0 +1,181 @@
+package org.sdase.commons.server.cloudevents;
+
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import java.net.URI;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+/**
+ * @param <T> the type of the data element of this CloudEvent
+ * @see <a href="https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md">1.0.2
+ *     Specification</a>
+ */
+public class CloudEventV1<T> extends BaseCloudEvent {
```

Comment:
Maybe we can provide some @JsonSchemaExamples with @JsonSchemaFormat and mark the required attributes also within the documentation: @JsonProperty(required = true)

Reply:
The required attributes are derived from the validation annotations `NotNull` / `NotEmpty`. Have a look at the generated example documentation that I just added to the PR.